src/main/java/htsjdk/samtools/util/BlockCompressedInputStream.java

```diff
@@ -50,6 +50,13 @@
  * c.f. http://samtools.sourceforge.net/SAM1.pdf for details of BGZF format
  */
 public class BlockCompressedInputStream extends InputStream implements LocationAware {
+
+    public final static String INCORRECT_HEADER_SIZE_MSG = "Incorrect header size for ";
+    public final static String UNEXPECTED_BLOCK_LENGTH_MSG = "Unexpected compressed block length: ";
+    public final static String PREMATURE_END_MSG = "Premature end of ";
+    public final static String CANNOT_SEEK_STREAM_MSG = "Cannot seek on stream based file ";
```

Comment:
Don't want a colon because this message will not be followed by a file name, the file name is not in the stream.